content/ch-states/case-for-quantum.ipynb

```diff
@@ -59,7 +59,7 @@
     "  <summary>Definition: Big O notation (Click to expand)</summary>\n",
     "For some example functions $f(x)$ and $g(x)$ and parameter $x$, the statement $f(x) = O(g(x))$  means that there exists some finite numbers $M>0$ and $x_0$ such that\n",
```

Comment:
Could we update this to "exist" rather than exists, since we are referring to numbers?

Reply:
Addressed.